site/src/component/RecentOfferings/RecentOfferings.tsx

```diff
@@ -66,14 +110,24 @@ const RecentOfferings: FC<RecentOfferingsProps> = (props) => {
           {/* only display the last 4 years of offerings */}
           {Object.entries(offerings)
             .slice(0, 4)
-            .map(([year, quarters]) => (
-              <tr key={year}>
-                <td>{year}</td>
-                {quarters.map((offered, index) => (
-                  <td key={index}>{offered ? <CheckIcon style={{ fontSize: 20 }} /> : null}</td>
-                ))}
-              </tr>
-            ))}
+            .map(([year, quarters]) => {
+              return (
+                <tr key={year}>
+                  <td>{year}</td>
+                  {quarters.map((offered, index) => {
```

Comment:
you should be able to make this and the `return <tr .../>` a direct arrow return `(...) => <RecentOfferingsQuarter .../>`, but it's not that big of a deal